content/articles/account-securing.md

```diff
@@ -35,6 +35,11 @@ If you've noticed fraudulent activity, or suspect someone has taken over your ac
 - We strongly recommend enabling [multi-factor authentication (MFA)](/articles/multi-factor-authentication/) for an extra layer of security. MFA is available free for every subscription.
 - [Enforce MFA](/articles/multi-factor-authentication-enforcement/) for every team member managing domains on your account.
 - Download and secure your recovery codes.
+- Accounts that don't have MFA enabled are required to change their password every 3 months. The new password cannot be any of the last 5 passwords used.
+
+## Forgotten password
+
+Find out how to request for a password reset [here](/articles/forgot-password).
```

Comment:
"Find out how to request a password reset here"

You can drop the "for a" to have this worded more clearly.